CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ cr_add_subproject (nanomsg
 )
 
 cr_add_subproject (wingetopt PATH dependencies/wingetopt CMAKE IF MSVC)
+cr_add_subproject (debugbreak PATH dependencies/debugbreak MAKE)
```

Comment:
Don't add it as a subproject, it's a header only library that does not need building.